### ISSUE TITLE
model.load_weights is not working

### ISSUE THREAD

**Issue by JackWang0522** [SciSharp/TensorFlow.NET#814]
I have a sequential model built in F#, and want to load the weights of a model trained in Python.

However, when I call
`model.load_weights("path")`

The weights in the model doesn't change at all.

I really hope we can load a model directly from a h5 file in the future, or at least get the load_weights working.

Thanks

**Comment by mikesneider** [comment]
HI @JackWang0522 did you solve by yourself this issue? I got an error too, but my error is trying to import the assembly HDF5CSharp, Version=1.1.10.500.

**Comment by nomis51** [comment]
I also get an error while loading h5 file using `model.load_weights`. I'm getting an exception `NotImplementException` after calling `executing_eagerly_outside_functions`. Not really sure what's happening. My model was also trained in Python and it runs fine in Python.

**Comment by lionwu** [comment]
There are still some problems below:

1. after invoke model.fit(), model.weights is a empty list
2. model.save_weights() is work, but when I rebuild model and load weight and predict again, model.load_weights() is not work correctly.


<pre><code>
//training model
Sequential model = BuildModel();
model.compile(optimizer: keras.optimizers.Adam(learning_rate), loss: keras.losses.MeanSquaredError(), metrics: new[] { "accuracy" });
model.fit(dataSet.X, dataSet.Y, batch_size: 2, epochs: 6, verbose: 0);//training model            
model.save_weights(savePath); //.NETCore work, .netframework not work
Predict(model);
//Console echo:  [Predict] 7.098797

//load weight and predict
model = BuildModel();
model.compile(optimizer: keras.optimizers.Adam(learning_rate), loss: keras.losses.MeanSquaredError(), metrics: new[] { "accuracy" });
model.load_weights(savePath);            
Predict(model);
//Console echo:  [Predict] 1.1410681

</code></pre>
